users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -131,16 +147,19 @@ def apply_codecs(audio, sample_rate, random_state, codecs):
         for codec in codecs:
             if codec.get("encoding") == "ULAW":
                 prob = codec.pop("prob", 1.0)
-                min_value = codec.pop("minimum", 255)
-                max_value = codec.pop("maximum", 255)
+                min_value = codec.pop("minimum", 1)
+                max_value = codec.pop("maximum", 1)
```

Comment:
What is the motivation for changing the default to 1?

Reply:
1 does not change the audio and is in my opinion the expected default if no value is given.

Reply:
I think it does, right?

Reply:
My bad. The change was not hear able or visible. I guess the correct value would be 0 but this is not a viable value. I am not sure what other value I should choose.

Reply:
I think 255 makes sense since it is the default value used for mu-law encoding. It is maybe not a good value for us, but we can overwrite it in any case.

Reply:
It seems the most obvious value to expect from a default imho.

Reply:
Just as the default for maximum or for minimum as well?
1 seems like the best option for minimum as the smallest possible change with an integer.

Reply:
I would also do 255 for minimum. Because then you get the default mu-law encoding if you don't specify anything.

Reply:
It won't matter too much because you'll specify the values anyway for all experiments I guess. But this seems like the most natural default.